app/models/katello/docker_meta_tag.rb

```diff
@@ -49,6 +53,10 @@ def schema2_manifest
       schema2.try(:docker_manifest)
     end
 
+    def self.with_uuid(ids)
```

Comment:
I am mixed about whether you should do something like
"alias_method ...." . Feel free to add